packages/material-ui/src/Box/Box.d.ts

```diff
@@ -1,16 +1,48 @@
 import * as React from 'react';
-import { SxProps } from '@material-ui/system';
+import {
+  borders,
+  ComposedStyleFunction,
+  display,
+  flexbox,
+  grid,
+  palette,
+  positions,
+  shadows,
+  sizing,
+  spacing,
+  typography,
+  PropsFor,
+  SxProps,
+} from '@material-ui/system';
 import { OverrideProps, OverridableComponent } from '../OverridableComponent';
 import { Theme } from '../styles/createMuiTheme';
 
+export type BoxStyleFunction = ComposedStyleFunction<
```

Comment:
Does this the already exist in the system package? If not, should be it created?

Reply:
Ok, no need for new type, I just used the value directly in the generic